lib/membrane/telemetry/telemetry.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Membrane.Telemetry do
+  @moduledoc """
+  Defines basic telemetry event types used by Membrane.Core that can be consumed by attaching to [Telemetry Package](https://hex.pm/packages/telemetry).
```

Comment:
If you really want to mention module then it should be wrapped with backticks (\`\`). However I think we can be more general here and say about the whole package:
```suggestion
  Defines basic telemetry event types used by Membrane's Core that can be consumed by attaching to [Telemetry Package](https://hex.pm/packages/telemetry).
```